docs/lib/datastore/schema-updates.md

```diff
@@ -24,6 +26,9 @@ type Post @model {
 
 This will evaluate the changes and create a versioned hash if any changes are detected which impact the underlying on-device storage structure. For example, types being added/deleted or fields becoming required/optional. DataStore evaluates this version on startup and if there are changes the local items on device will be removed and a full sync from AppSync will take place if you are syncing with the cloud.
 
-### Migrations
+## Migrations
+
+Local migrations on device are not currently supported. If you are syncing with the cloud the structure and items of that data in your DynamoDB table will not be touched as part of this process.
```

Comment:
We might need to describe what a local migration is before saying that it is not supported?